test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1

```diff
@@ -127,6 +127,7 @@ Describe "Select-String" -Tags "CI" {
 
 	It "Should return ParameterBindingException when -Raw and -Quiet are used together" {
 		{ $testinputone | Select-String -Pattern "hello" -Raw -Quiet -ErrorAction Stop } | Should -Throw -ExceptionType ([System.Management.Automation.ParameterBindingException])
+        }
```

Comment:
This brace seems out of place?  I think this whole file should be reformatted so the intention is correct making it easier to understand the braces.

Reply:
Maybe that would be better done in a quick follow-up PR so we can keep track of the diffs and so accidental changes are made?

Reply:
The reformat can be a single commit.  Then the fix of the brace a separate commit.

Reply:
That test isn't touched in this PR. My last commit just undid an accidental deletion of a brace.

Reply:
@TylerLeonhardt perhaps you can just add a commit after reformatting the tests?

Reply:
What's wrong with the formatting exactly? I don't see anything wrong with the changes in this PR.

Reply:
Not this PR specifically, but if you [view the whole file](https://github.com/PowerShell/PowerShell/blob/46e62fd2dd88f36b765c194a67cdcf18049b715e/test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1), the indentation is all over the place.  Makes it hard to review the code.  I still can't tell how that missing brace fixed the code since the indentation isn't consistent.

Reply:
The indentation issue is because about half the file is indented with tabs instead of spaces. Github renders a tab as 8 spaces, whereas most editors commonly only render it with 4 spaces (though this is configurable).

I think VS Code has a convert tabs to spaces option in its command palette to get rid of the tabs, and then you can do a quick skim over the file to fix any remaining issues.

Reply:
Ok, I see.  For this repo, we've standardized on spaces.

Reply:
done.